ktx/lifecycle-livedata-ktx/README.md

```diff
@@ -10,11 +10,10 @@ Extended set of extensions for dealing with `LiveData`.
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 
-
 - [Installation](#installation)
 - [Usage](#usage)
   - [`LiveData` delegate](#livedata-delegate)
-  - [Events Queue](#events-queue)
+  - [ViewModelEvents](#viewmodelevents)
```

Comment:
I think `ViewModelEvents` should be removed from this README. It would be great to add a note about this removal to the changelog of `lifecycle-livedata-ktx`